flake.nix

```
@@ -243,6 +243,7 @@
           changelog-d;
         default = self.packages.${system}.nix;
       } // lib.concatMapAttrs
+        # We need to "flatten" packages we care about to pass `flake check`.
```

```suggestion
        # We need to "flatten" package sets to pass `flake check`.
```

- sets
- what if we don't care?

---

I meant we're not doing all of `nixpkgsFor.*` but filtering down to a select few packages. I don't like my phrasing either, is there a better way?

---

```suggestion
        # We need to flatten recursive attribute sets of derivations to pass `flake check`.
```